src/twisted/protocols/test/test_tls.py

```diff
@@ -1851,17 +1877,6 @@ def acceptableProtocols(self):
 class IProtocolNegotiationFactoryTests(TestCase):
```

Comment:
Isn't this the TestCase (Or at least one of its test methods) that was being deleted and then fixed later in https://twistedmatrix.com/trac/ticket/10067 ?